src/utils_general/statistics.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+import pandas as pd
+from scipy.interpolate import interp1d
+
+from scipy.stats import genextreme as gev
+import matplotlib.pyplot as plt
+
+def _get_return_period_function_analytical(
```

Comment:
The method names should not begin with an underscore, as that is used to denote private methods (i.e. not used outside their own class / module)